### DISCUSSION TITLE
Application hidden with F-Droid Basic ?

### DISCUSSION THREAD

**Discussion by pled** [organicmaps/organicmaps#9973] [Q&A] [answered]
Hi,
I just noticed that Organic Maps is not displayed when searching for the app using F-Droid Basic. I had to go to Parameters - Include applications with anti-functions, then check "Tethered Network Services" and "Not Secured"" (actually the 2 unchecked by default) to get Organic Maps displayed. 
I guess this is not expected ?
Running GrapheneOS Android 15 and F-Droid Basic 1.12.1.

**Comment by pastk** [comment, marked as answer]
Yeap, it's an issue on F-Droid's side, see https://gitlab.com/fdroid/fdroidclient/-/issues/2850
